v3/data/popup/index.js

```diff
@@ -281,9 +281,37 @@ function msg(msg) {
   toast.textContent = msg;
   window.setTimeout(() => toast.textContent = '', 2000);
 }
+// get random elements
+function getRandomElement(array) {
+  console.log(array);
+  return array[Math.floor(Math.random() * array.length)];
+}
+
+// gets random browser and os
+function getRandomBrowserAndOS() {
```

Comment:
The random UA, must be selected from well-known OS and browsers. Selecting a random OS breaks many websites.